tests/manage/z_cluster/cluster_expansion/test_add_capacity.py

```diff
@@ -95,3 +98,24 @@ def test_add_capacity_pre_upgrade(self, reduce_and_resume_cluster_load):
         Test to add variable capacity to the OSD cluster while IOs running
         """
         add_capacity_test()
+
+
+@tier1
+@skipif_no_lso
+@skipif_external_mode
+@skipif_bm
+@skipif_aws_i3
+@skipif_ocs_version("<4.7")
+class TestAddCapacityLso(ManageTest):
+    """
+    Automates adding variable capacity to the lso cluster
+
+    """
+
+    def test_add_capacity_lso(self):
+        """
+        Test to add capacity to the lso cluster
+
+        """
+        new_nodes = add_new_node_and_label_upi()
+        add_capacity_lso(new_node=new_nodes[0])
```

Comment:
Need to cover 'add_capacity' without adding a new node. That is, add disk to an existing node. Is this covered somewhere else?

Reply:
I tested 'add_capacity' with adding a new node
![image](https://user-images.githubusercontent.com/61982127/108712017-db153180-751e-11eb-8818-192d35de9b41.png)

I **did not test**  'add_capacity' without adding a new node:
![image](https://user-images.githubusercontent.com/61982127/108712137-0435c200-751f-11eb-8005-0f02b054bb11.png)

Reply:
@hnallurv 
I will add support for 'add_capacity' without adding a new node